packages/ripple-address-codec/package.json

```diff
@@ -20,7 +20,7 @@
   "prepublish": "tsc -b",
   "prepublishOnly": "tslint -b ./ && jest",
   "scripts": {
-    "build": "tsc --build tsconfig.build.json",
+    "build": "tsc --build tsconfig.build.json --force",
```

Comment:
Why the `--force`? That seems like bad practice. Is this just an artifact of some testing?

Reply:
After updating to the latest typescript incremental builds seems broken and locally will not compile when running `npm run build` and putting the code correct folder.  It will put it in `dist/src/*` instead of `dist/*`. I will try to fix this locally.  The only other way around it is to manually delete tsbuilsldinfo files.

Reply:
I figured out it wasnt deleting tsbuilsldinfo files because I needed to update the `clean` script.  This meant after running clean the buildinfo said there were files but they had just been deleted.  I will create a PR that fixes this rather than adding `--force`.

Reply:
Will be corrected in #2468.